### PR TITLE
check if name is not empty before constructing message.

### PR DESCRIPTION
**Pull request by najibraihan**

### CODE REVIEW

File: agent/result_parser.py [hunk 1]
```diff
@@ -33,26 +33,29 @@ def parse_results(results: whois.parser.WhoisCom) -> Iterator[Dict[str, Any]]:
...
-            ),
-            "contact_name": contact_name,
-            "dnssec": get_list_from_string(scan_output_dict.get("dnssec", "")),
-        }
-        for field in OPTIONAL_FIELDS:
-            if field in scan_output_dict:
-                value = scan_output_dict[field]
-                output[field] = _format_str(value) if value is not None else value
-        yield output
+        if name != "":
```
Comment (amine3) [inline agent/result_parser.py:36]: Missing unit test to check the behavior if name is empty string

**Review by amine3 — CHANGES_REQUESTED** [review]

**Review by 3asm — DISMISSED** [review]

**Review by amine3 — APPROVED** [review]

**Review by 3asm — APPROVED** [review]
